README.md

```diff
@@ -13,21 +13,113 @@ You will first install Pypush onto your machine. *Please keep in mind that you w
 
-1. Enable USB debugging/ADB on your phone. There are multiple online guides that guide you through this based on your phone.
-2. Install the APK. The message link containing the APK is located [here](https://discord.com/channels/1130633272595066880/1145177252015915080/1153070972090470481), and the GitHub repository is [here](https://github.com/JJTech0130/PNRGatewayClientV2).
-3. Grant SMS permissions. This will be in the app info page, and on the newer version, there should be a button in the app that does this for you.
-4. Connect your phone to the same WiFi network as your host PC, and open the app.
+1. Install the APK. The message link containing the APK is located [here](https://discord.com/channels/1130633272595066880/1145177252015915080/1153070972090470481), and the GitHub repository is [here](https://github.com/JJTech0130/PNRGatewayClientV2).
+2. Open the app.
+3. Grant SMS permissions by clicking the button and approving the permissions.
+4. Connect your phone to the same WiFi network as your host PC, and open the app (unless using Android: Termux, then it doesn't matter what network)
 
 ### Pypush
 Once you have the PNRgateway app installed on your phone, open it so it is displaying your IP address as you will need it for the next steps. 
 
-Use one of the automated installers for your operating system: [Windows](https://github.com/JJTech0130/pypush/blob/bacefed8b8eb78d5d3f295be5304830665464a04/windows_installer.ps1) or [MacOS/Linux](https://github.com/JJTech0130/pypush/blob/bacefed8b8eb78d5d3f295be5304830665464a04/unix_installer.sh)
+Use one of the automated installers for your operating system: [Windows](https://github.com/JJTech0130/pypush/blob/bacefed8b8eb78d5d3f295be5304830665464a04/windows_installer.ps1), [MacOS/Linux](https://github.com/JJTech0130/pypush/blob/bacefed8b8eb78d5d3f295be5304830665464a04/unix_installer.sh), or [Android Termux](./termux_installer.sh).  
 
-For Windows open up PowerShell and navigate to your downloads folder `cd Downloads` and then execute the installer `.\windows_installer.ps1` and follow the prompts. When initial registration has completed execute the file `windows_reregister.ps1` to handle reregistration. This file will reregister your number 5 minutes before registration expires and you must keep the PowerShell window open. Length of registration will gradually increase. 
+If you need help or run into errors please reach out on our [Discord](https://discord.gg/BtSbcExKJ9) server.
 
-For MacOS/Linux open up your terminal and navigate to your downloads folder `cd Downloads` or similar. Make the script executable by executing `chmod +x unix_installer.sh`. Execute the script `./unix_installer.sh`. Upon completion a shell script is created called `reregister.sh`. Execute this script in your terminal `./reregister.sh`. This file will reregister your number 5 minutes before registration expires and you must keep the terminal window open. Length of registration will gradually increase.
```

Comment:
Why is this being deleted?

Reply:
It's just converted into a numbered list of instructions with the same steps.